script.js

```diff
@@ -1,111 +1,76 @@
-const inputTitle = document.querySelector('#title');
-const inputAuthor = document.querySelector('#author');
-const addBtn = document.querySelector('.add-but');
+// eslint-disable-next-line max-classes-per-file
 const booksSection = document.querySelector('.booksSection');
-
-let books = [];
-
-const renderBooks = () => {
-  booksSection.innerHTML = '';
-  books.forEach((book) => {
-    const newBook = document.createElement('div');
-    newBook.classList.add('book');
-    newBook.innerHTML = `
-      <p>${book.title} by ${book.author}</p>
-      <button class="remove-but">Remove</button>
-    `;
-    booksSection.appendChild(newBook);
-    localStorage.setItem('books', JSON.stringify(books));
- });
-};
-
-window.addEventListener('DOMContentLoaded', () => {
-  const storageBooks = JSON.parse(localStorage.getItem('books'));
-  if (storageBooks) {
-    books = storageBooks;
-    renderBooks();
+class First {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
   }
-});
+}
 
-const validateForm = () => {
-  if (inputTitle.value === '' || inputAuthor.value === '') {
-    const alertDiv = document.createElement('div');
-    alertDiv.classList.add('alert');
-    alertDiv.innerHTML = 'Please fill in all fields';
-    document.body.appendChild(alertDiv);
-    setTimeout(() => {
-      alertDiv.remove();
-    }, 1000);
-    return false;
+class Second {
+  constructor() {
+    this.list = JSON.parse(localStorage.getItem('data')) || [];
   }
-  return true;
-};
 
-const addBook = (e) => {
-  e.preventDefault();
-  if (validateForm()) {
-    const title = inputTitle.value;
-    const author = inputAuthor.value;
-    const book = {
-      title,
-      author,
-    };
-    books.push(book);
-    localStorage.setItem('books', JSON.stringify(books));
-    inputAuthor.value = '';
-    inputTitle.value = '';
+  add(data) {
+    if (data.title === '' || data.author === '') {
+      // eslint-disable-next-line no-alert
```

Comment:
The main objective of the linters is to prevent programming errors, bugs, stylistic errors, and suspicious constructs. Kindly remove this line of code and fix the issue flagged by the linters tool. :+1: :smiley: :smiley: